test/elixir_bench/benchmarks/measurement_test.exs

```diff
@@ -0,0 +1,85 @@
+defmodule MeasurementTest do
+  alias ElixirBench.Benchmarks.Measurement
+  use ElixirBench.DataCase
+
+  @attrs %{
+    "average" => 465.8669101807624,
+    "ips" => 2146.5357984150173,
+    "maximum" => 13092.0,
+    "median" => 452.0,
+    "minimum" => 338.0,
+    "percentiles" => %{"50" => 452.0, "99" => 638.0},
+    "run_times" => [],
+    "sample_size" => 10677,
+    "std_dev" => 199.60367678670747,
+    "std_dev_ips" => 919.6970816229071,
+    "std_dev_ratio" => 0.4284564377179282,
+    "mode" => nil
+  }
+
+  describe "changeset/2" do
+    test "cast required fields" do
+      changeset = Measurement.changeset(%Measurement{}, @attrs)
+
+      assert changeset.valid?
+
+      assert [
+               :average,
+               :ips,
+               :maximum,
+               :median,
+               :minimum,
+               :percentiles,
+               :run_times,
+               :sample_size,
+               :std_dev,
+               :std_dev_ips,
+               :std_dev_ratio
+             ] = Map.keys(changeset.changes)
+    end
```

Comment:
not sure we need this test, we more or less just repeat the list of fields. We should be fine without it I think :)

Reply:
it doesn't cast the `:mode` attribute, the only field that is not required. But I agree that this is kind of a weak test, this case is someway covered in the next test cases where I do `mode = nil`. Will remove this.